cronjob.yml

```diff
@@ -0,0 +1,32 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: minio-amundsen-loader
```

Comment:
We should still be opinionated about namespace -- can we add `namespace: amundsen` here?